README.md

```diff
@@ -1,3 +1,5 @@
+![img](https://github.com/richardbmx/warthog/blob/master/github.png?raw=true)
```

Comment:
Once you make change below, you should be able to refer to this relatively `./images/logo.png?raw=true`  Then you don't need to refer to your fork.